aws-msk/config.yml

```
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: template-quickstart
```

Please update this with a hyphenated name. Something like:
```suggestion
name: aws-msk
```